x/crosschain/keeper/cctx.go

```diff
@@ -48,15 +59,42 @@ func (k Keeper) SetCctxAndNonceToCctxAndInboundHashToCctx(
 		in.CctxIndex = append(in.CctxIndex, cctx.Index)
 	}
 	k.SetInboundHashToCctx(ctx, in)
+}
 
-	if cctx.CctxStatus.Status == types.CctxStatus_Aborted && cctx.InboundParams.CoinType == coin.CoinType_Zeta {
+// updateZetaAccounting updates the zeta accounting with the amount of zeta that was locked in an aborted cctx
+func (k Keeper) updateZetaAccounting(
+	ctx sdk.Context,
+	cctx types.CrossChainTx,
+) {
+	if cctx.CctxStatus.Status == types.CctxStatus_Aborted &&
+		cctx.InboundParams.CoinType == coin.CoinType_Zeta &&
+		cctx.CctxStatus.IsAbortRefunded == false {
 		k.AddZetaAbortedAmount(ctx, GetAbortedAmount(cctx))
 	}
 }
 
+// setNonceToCCTXMapping updates the mapping between a nonce and a cctx index if the cctx is in a PendingOutbound or PendingRevert state
+func (k Keeper) setNonceToCCTXMapping(
```

Comment:
```suggestion
func (k Keeper) updateNonceToCCTX(
```
We update an existing value
`NonceToCCTX` is the name of the variable, no need for mapping suffix